tests/foreman/ui/test_remoteexecution.py

```diff
@@ -0,0 +1,227 @@
+"""Test class for Remote Execution Management UI"""
+from robottelo.decorators import stubbed, tier1, tier2
+from robottelo.test import UITestCase
+
+
+class TestRemoteExecution(UITestCase):
+    """Test class for remote execution feature"""
+
+    @stubbed()
+    @tier1
+    def test_positive_create_simple_job_template(self):
+        """Create a simple Job Template
+
+        @Feature: Remote Execution
+
+        @Steps:
+
+        1. Navigate to Hosts -> Job Templates
+        2. Enter a valid name
+        3. Populate the template code
+        4. Navigate to the job tab
+        5. Enter a job name
+        6. Click submit
+
+        @Assert: The job template was successfully created
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier1
+    def test_positive_create_job_input_template(self):
```

Comment:
It seems more like `test_positive_create_job_template_input`, but not sure on 100%